media/ivf.ksy

```diff
@@ -0,0 +1,67 @@
+meta:
+  id: ivf
+  file-extension: ivf
+  endian: le
+  title: VP8 raw file format
+  ks-version: 0.7
+  license: CC0-1.0
+doc-ref: https://wiki.multimedia.cx/index.php/IVF
+seq:
+## header start
+  - id: magic1
+    contents: DKIF
+    doc: Magic Number of IVF Files
+  - id: version
+    type: u2
+    doc: This should be 0
+  - id: headerlen
+    type: u2
+    doc: Normally the header length is 32 byte
```

Comment:
Could you make this `len_header` to match [style guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id) recommendation?